point-addition/src/lib.rs

```diff
@@ -1,15 +1,26 @@
+//! A library for converting two-party shares of an elliptic curve (EC) point's x-coordinate under
```

Comment:
iiuc, the 2 parties each start with an EC point. These 2 EC points are additive shares of another EC point. So, I took a stab at simplifying this description:

A secure two-party computation (2PC) library for converting additive shares of an elliptic curve (EC) point into additive shares of said point's x-coordinate. <Optionally an extra sentence mentioning that the x-coordinate is a field element>